src/itkConfigDevelopment.ts

```diff
@@ -1,4 +1,4 @@
-const version = '1.0.0-b.18'
+const version = '1.0.0-b.38'
```

Comment:
Would it make sense to insert a placeholder here instead? And have the build script find/replace it?

```suggestion
const version = '<%PKG_VERSION%>'
```

Reply:
I prefer this approach, since I don't think there's any bit of internal code that actually cares about the format of the `version` variable. Either a regular string.replace call works fine, or the use of lodash templates.

Reply:
Yes, agreed.